packages/fx-core/src/plugins/resource/frontend/dotnet/constants.ts

```diff
@@ -65,6 +65,9 @@ export class DotnetPathInfo {
     framework = DotnetPluginInfo.defaultFramework,
     runtime = DotnetPluginInfo.defaultRuntime
   ): string => path.join(workingPath, "bin", "Release", framework, runtime, "publish");
+
+  static readonly bicepTemplateFolder = (templateFolder: string) =>
+    path.join(templateFolder, "plugins", "resource", "webapp", "bicep");
```

Comment:
Nit: Although checked the point in related comment below, I still recommend using `dotnet` instead of `webapp`, which is based on to simplify concept for readability and to follow existing style/contract, I mean, (bot/simpleauth/tab used (webapp/webapp/storage) for provision/deploy, but no webapp plugin/storage plugin existed since we may treat them as implementation choices).

Reply:
1. In v3 design, there will be a dotnet plugin for scaffolding and building dotnet project, and a webapp plugin for provisioning resources and deploying code. They are mapping to current dotnet plugin's scaffold and provision, deploy.
I would like to distinguish them in advance.
2. We used to naming plugin by capability or component (bot/simpleauth/function), but `dotnet` is different from those. It is the runtime of the project. It is wired for me to see a dotnet/bicep folder.